### PR TITLE
#758 Add Echorouk

### PR DESCRIPTION
**Pull request by link8k**
from https://www.ennaharonline.com/live/
@themak1985 requested in #758

https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8
https://streamtest.in/logs/http-echorouk-live-tv-dzsecurity-net-8081-echo-echorouknews-playlist-m3u8

### CODE REVIEW

File: channels/dz.m3u [hunk 1]
```diff
@@ -5,3 +5,9 @@ http://178.132.3.162:88/canal_algerie/video.m3u8?token=test
 http://numidiatv-live.dzsecurity.net:8081/entv/EnnaharTV/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://upload.wikimedia.org/wikipedia/en/thumb/c/cc/Ennahar_TV.svg/330px-Ennahar_TV.svg.png" group-title="",Ennahar TV (360p)
 http://numidiatv-live.dzsecurity.net:8081/entv/EnnaharTV_SD/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="",Echorouk TV
+#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net
+https://echorouk-live-tv.dzsecurity.net/echo/EchoroukTV/playlist.m3u8
```
Comment (adamo331) [inline channels/dz.m3u:10]: not working

File: channels/dz.m3u [hunk 2]
```diff
@@ -5,3 +5,9 @@ http://178.132.3.162:88/canal_algerie/video.m3u8?token=test
...
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="" group-title="News",Echorouk News
+#EXTVLCOPT:http-referrer=https://echorouk.dzsecurity.net
+https://echorouk-live-tv.dzsecurity.net/echo/EchoroukNews/playlist.m3u8
```
Comment (adamo331) [inline channels/dz.m3u:13]: not working

**Comment by LaneSh4d0w** [comment]
DZ-Security got some weird things happening. It exists... but it's got some security issues due to some authenticity problems.

Funny thinking about a society that got "Security" inside of it.

**Comment by LaneSh4d0w** [comment]
@link8k Change those with this, less techy and works : 

Echourouk TV : http://echorouk-live-tv.dzsecurity.net:8081/echo/EchoroukTV/playlist.m3u8
Echourouk News : http://echorouk-live-tv.dzsecurity.net:8081/echo/EchoroukNews/playlist.m3u8

**Comment by link8k** [comment]
Thanks @LaneSh4d0w @adamo331  Fixed

**Review by freearhey — APPROVED** [review]

**Pull request merged**